.github/workflows/update_release_notes.yml

```diff
@@ -0,0 +1,68 @@
+name: Update release notes
+
+on:
+  push:
+    branches:
+      - 'release-notes'
+
+jobs:
+  fetch_version_name_release_notes:
```

Comment:
What do you think of changing the name to just `fetch_version_name` and omit `release_notes` to keep it more simple?